src/site/markdown/getting-started.md

```diff
@@ -129,17 +129,24 @@ For the Context Class Loader case this would mean the following dependencies are
 
 ## Run
 
-Calling `executeGoals` runs Maven with the given goals or phases and optionally some additional environment variables. It throws a `VerificationException` in case the execution is not successful (e.g. binary not found or exit code > 0). It is either using a forked JVM or is executed in the same JVM depending on the configuration.
+Calling `execute` runs Maven with the given Verifier configuration like, cli arguments, environment variables. 
+We must provide at lease one cli argument for executing.
```

Comment:
```suggestion
At least one CLI argument must be given which specifies either the Maven phase(s) and/or the goal(s) to execute.
```

Reply:
Do we really need that? What if `/project/build/defaultGoal` is set?

Reply:
must be provided

Reply:
Maybe:

``
At least one CLI argument should be provided which specifies either the Maven phase(s) and/or the goal(s) to execute unless project has default goal.
``

Reply:
...has a default goal.

Sounds good to me.